_docs/cardano/2017-02-02-transaction-fees.md

```diff
@@ -0,0 +1,68 @@
+---
+layout: default
+title: Transaction Fees
+permalink: /cardano/transaction-fees/
+group: cardano
+---
+
+# Transaction Fees in Cardano SL
+
+## Motivation
+
+There are two main reasons why transaction fees are needed for Cardan SL:
+
+1.  People who run full Cardano SL nodes spend time, money and effort to run the protocol, for which they should
+    be compensated and rewarded. In contrast to Bitcoin, where new currency is created with each mined block,
+    in Cardano SL, transaction fees are the only source of income for participants in the protocol.
+2.  The second reason is the prevention of DDoS (Distributed Denial of Service) attacks. In a DDoS attack, an attacker
+    tries to flood the network with dummy transactions, and if he has to pay a sufficiently high fee for each of those
+    dummy transactions, this form of attack will become prohibitively expensive for him.
+
+## How transaction fees work
+
+Whenever somebody wants to transfer an amount of Ada, some minimal fees are computed for that transaction. In order for
+the transaction to be valid, these minimal fees have to be included (although the sender is free to pay higher fees if
+he so wishes). All transaction fees are collected in a virtual pool and then later distributed amongst participants in
+the Cardano SL protocol.
```

Comment:
It's not implemented now. Fees just disappear.

Reply:
I see that later you write about it, but I think it's better to write about it here as well, because otherwise reader may be confused until they read the last part.